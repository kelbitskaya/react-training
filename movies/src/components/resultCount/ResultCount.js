import React from 'react';
import PropTypes from 'prop-types';

export default function ResultCount(props) {
  const { count } = props;
  return (
    <div className="result-count">
      <strong>{count}</strong>
      <span> movies founded</span>
    </div>
  );
}

ResultCount.propTypes = {
  count: PropTypes.number.isRequired,
};
