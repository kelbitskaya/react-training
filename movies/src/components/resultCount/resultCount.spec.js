import React from 'react';
import ResultCount from './ResultCount';

describe("ResultCount component", ()=>{
  it("should render ResultCount with props", ()=> {
    const component = shallow(<ResultCount count={1} />);
    expect(component).toMatchSnapshot();
  });
});
