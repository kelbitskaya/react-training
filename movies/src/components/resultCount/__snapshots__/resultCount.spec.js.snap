// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultCount component should render ResultCount with props 1`] = `
<div
  className="result-count"
>
  <strong
    className="test"
  >
    1
  </strong>
  <span
    className="result-count__text"
  >
     movies founded
  </span>
</div>
`;
